multiqc/modules/jellyfish/jellyfish.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse results from jellyfish  """
+
+from __future__ import print_function
+
+from collections import OrderedDict
+import logging
+from multiqc import config
+from multiqc.plots import linegraph, bargraph
+from multiqc.modules.base_module import BaseMultiqcModule
+
+
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='jellyfish', anchor='jellyfish',
+        href="http://www.cbcb.umd.edu/software/jellyfish/",
+        info="JELLYFISH is a tool for fast, memory-efficient counting of k-mers in DNA.")
+
+        self.jellyfish_data_all = dict()
+        for f in self.find_log_files('jellyfish', filehandles=True):
+            self.parse_jellyfish_data(f)
+        
+        #clean histogram to prevent long flat tail
+        self.clean_jellyfish_data()
+        
+        if len(self.jellyfish_data) == 0:
+            log.debug("Could not find any data in {}".format(config.analysis_dir))
+            raise UserWarning
+            
+        log.info("Found {} reports".format(len(self.jellyfish_data)))
+        
+        self.intro += """Jellyfish k-mer plots for library complexity. 
```

Comment:
This is pretty long.. Maybe better in `self.add_section` under the `help` attr? Will display in a similar way currently, but planning to make the styling better in the future so that it's hidden by default unless the user clicks a 'Help' button..